drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/collectioneditor/CollectionView.java

```diff
@@ -147,13 +151,11 @@
 
     SpanElement getPropertyTitle();
 
-    ButtonElement getAddItemButton();
-
-    ButtonElement getCancelButton();
-
-    ButtonElement getRemoveButton();
-
-    ButtonElement getSaveButton();
+    /**
+     * It enables/disables buttons group which manage the Collection Editor
+     * @param enable
+     */
+    void disableCollectionEditorButtons(boolean enable);
```

Comment:
I feel that confusing-parameter name should be disable according to the function name.
Enable of disableness is hard to imagine)

Reply:
@dupliaka You're right, I forgot to update the name.